be1-go/cli/organizer/organizer.go

```diff
@@ -44,36 +44,51 @@ func OrganizerServe(context *cli.Context) error {
 	done := make(chan struct{})
 	go h.Start(done)
 
-	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-		organizerServeWs(h, w, r)
+	go orgCreateSocket(hub.WitnessSocketType, h, port)
+	orgCreateSocket(hub.ClientSocketType, h, port)
+
+	done <- struct{}{}
```

Comment:
I might be missing something, but why is this needed? And why does it matter if it's initialized before or after the socket creation?

Reply:
This is because "done <- struct{}{}" signals the end of the program and tells the organizer to stop serving messages (check organizerHub.start).

Reply:
Ohh, yes of course wasn't aware of that, thanks!